style.css

```diff
@@ -1,90 +1,219 @@
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
-    height: 100%;
-}
-
-.title-container {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    height: 22%;
-    background-color: antiquewhite;
-    border-radius: 10px 10px 10px 10px;
-}
-
-.right-section {
-    display: flex;
-    justify-content: flex-end;
-    align-items: center;
-}
-
-.preparation-container {
-    display: flex;
-    flex-direction: column;
-    background-color: antiquewhite;
-    width: 50%;
-    height: 85%;
-    border-radius: 10px 10px 10px 10px;
-}
-
-.preparation {
-    display: flex;
-    margin-left: 20px;
-}
-
-.ingredients-container {
-    display: flex;
-    flex-direction: column;
-    background-color: antiquewhite;
-    width: 55%;
-    height: 75%;
-    border-radius: 10px 10px 10px 10px;
-    margin-bottom: 10px;
-}
-
-.preparation-text {
-    margin-left: 20px;
-    margin-right: 20px;
-}
-
-.ingredients-text {
-    margin-left: 20px;
-}
-
-.ingredients {
-    margin-left: 20px;
-    display: flex;
-    justify-content: space-between;
-}
-
-.title-text {
-    margin-left: 20px;
-    margin-right: 20px;
-}
-
-h3{
-    margin-bottom:5px;
-}
-
-h2{
-    margin-bottom:10px;
-}
-
-img{
-    width:70px;
-    margin-right: 15px;
-    margin-top: 10px;
-}
+.main-container {
+    display: flex;
+    background-image: url('Dish.png');
+    background-size: 100%;
+    background-repeat: repeat;
+    flex-direction: row;
+    padding: 30px;
+    font-family: roboto, sans-serif;
+    width: 90vw;
+}
+
+.left-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    height: 100%;
+    width: 50%;
+}
+
+.title-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 10%;
+    width: 80%;
+    background-color: antiquewhite;
+    border-radius: 10px 10px 10px 10px;
+    margin-bottom: 20px;
+}
+
+.right-section {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    height: 100%;
+    width: 50%;
+}
+
+.preparation-container {
+    display: flex;
+    flex-direction: column;
+    background-color: antiquewhite;
+    width: 60%;
+    height: 35%;
+    border-radius: 10px 10px 10px 10px;
+    margin-bottom: 10px;
+}
+
+.preparation {
+    display: flex;
+    margin-left: 20px;
+}
+
+.ingredients-container {
+    display: flex;
+    flex-direction: column;
+    background-color: antiquewhite;
+    width: 50%;
+    height: 45%;
+    border-radius: 10px 10px 10px 10px;
+    margin-bottom: 20px;
+}
+
+.preparation-text {
+    margin-left: 20px;
+    margin-right: 20px;
+}
+
+.ingredients-text {
+    margin-left: 20px;
+}
+
+.ingredients {
+    margin-left: 20px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.title-text {
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-bottom: 20px;
+}
+
+h3{
+    margin-bottom:5px;
+}
+
+h2{
+    margin-bottom:10px;
+}
+
+img{

```

Comment:
Usually using a global tag selector is not a good idea, because you are changing every element of a specific type in the whole project. In bigger projects such a selector would be a pain in the ass.